msbuild/tests/Xamarin.iOS.Tasks.Tests/FrameworkListTest.cs

```diff
@@ -29,12 +29,12 @@ public void CheckFrameworkListFile (string frameworkListFile)
 
 			foreach (var assembly in frameworkListAssemblies) {
 				if (!installedAssemblies.Any (a => a.Name == assembly.Name))
-					ReportAssemblies (assembly, $"Incorrectly added assemblies in '{frameworkListFile}'");
+					ReportAssemblies (assembly, $"Incorrectly listed assemblies in '{frameworkListFile}'. Check if those assemblies/facades exist in the final SDK root folder.");
```

Comment:
I'd reword this as:
```
One or more assemblies listed in '{frameworkListFile}' were not found in the final SDK root folder. Update the list if an assembly was intentionally removed.
```

It makes it clearer what the expectation of the test is.